inst/examples/03-formats.Rmd

```diff
@@ -218,9 +231,235 @@ favicon: "favicon.ico"
+
+#### Writing a `bs4_book`
+
+The easiest way to get started writing a new `bs4_book` is to use the RStudio Project Wizard (*File > New Project > New Directory > Book project using bookdown*) and select `bs4_book` from the dropdown menu: 
+
+```{r fig.align='center', echo=FALSE, fig.alt="Screenshot of the RStudio Project Wizard for creating a new bs4_book project."}
+knitr::include_graphics('images/new-bs4-book.png', dpi = NA)
+```
+
+If you do not use RStudio or prefer a function, you can create the same project template with `bookdown::create_bs4_book()` from your R console. See `?create_bs4_book` for help.
+
+This theme is designed for books that use one chapter per page. 
+This means that each chapter is an `.Rmd` file, and each `.Rmd` file can contain one (and only one) chapter. 
+A chapter *must* start with a first-level heading: `# A chapter`, and can contain one (and only one) first-level heading. 
+
+Use second-level and lower-level headings within chapters like: 
+
+```md
+#   A chapter
+
+##  A section
+
+### A subsection
+```
+
+The first- and second-level headings will appear in that chapter's sidebar on the upper right as a "sticky" within-chapter table of contents that stays in place as you scroll down. In the sidebar, when a section is navigated to, subheadings will auto-expand such that third-level headings like "A subsection" are visible.
+
+The `index.Rmd` file is required, and is also your first book chapter. It will be the homepage when you render the book.
+
+A YAML header in `index.Rmd` for a `bs4_book` would look like this:
+
+```yaml
+---
+title: "A Minimal Book Example"
+author: "Jane Doe"
+date: "`r Sys.Date()`"
+site: bookdown::bookdown_site
+output: bookdown::bs4_book
+url: https://bookdown.org/janedoe/bookdown-demo
+cover-image: cover.png
+description: |
+  This is a minimal example of using the bookdown package to write a book.
+  The output format for this example is bookdown::bs4_book.
+---
+```
+
+#### Styling & customization
+
+This output format allows for easy customization of colors and fonts with the **bslib** package (<https://pkgs.rstudio.com/bslib/>).
+
+You can use the `theme` option to add a `primary` color in hexidecimal format (<https://en.wikipedia.org/wiki/Web_colors>), which will change the color of all links in your book and the background color of the footer.
+
+```yaml
+bookdown::bs4_book:
+  theme:
+    primary: "#0d6efd"   
+```
+
+For custom font settings, adding a `google:` keyword triggers [`sass::font_google()`’s](https://rstudio.github.io/sass/reference/font_face.html) ability to automatically import Google Font files (<https://fonts.google.com/>). By default, `google:` will bundle font files with your book, so that when you share it with someone else, the fonts are guaranteed to render, even without an internet connection (`local: false` imports files via URL instead of serving them locally). Here is an example YAML that changes the `base_font`, `heading_font`, and `code_font`:
```

Comment:
I think it would be better to mention `local: false` in a later paragraph, as this one is already information packed.